Samples/BackgroundBlur/BackgroundBlur/TransformAsync.cpp

```diff
@@ -0,0 +1,1212 @@
+#include "TransformAsync.h"
+#include <winrt/windows.media.h>
+#include <winrt/windows.foundation.h>
+#include <windows.graphics.directx.direct3d11.interop.h>
+#include <winrt/Windows.Graphics.DirectX.Direct3D11.h>
+#include <windows.graphics.imaging.interop.h>
+#include <windows.media.core.interop.h>
+#include "Helpers/common.h"
+#include <unknwn.h>
+#include <stdio.h>
+#include <wchar.h>
+#include <future>
```

Comment:
I would clean up code to use Std header will simplify the dependencies and include logic